### PR TITLE
CLOUDP-123294: Add Grafana example to the docs

### PR DESCRIPTION
**Pull request by fabritsius**

### CODE REVIEW

**Comment by fabritsius** [comment]
todo for tomorrow: rename clusters to deployments in the grafana example

**Review by priyolahiri — APPROVED** [review]

File: docs/metrics-via-grafana.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: "Test Atlas Operator Project-mongo-metrics"
+    scrape_interval: 10s
+    metrics_path: /metrics
+    scheme : https
+    basic_auth:
+      username: user
```
Comment (leo-ri) [inline docs/metrics-via-grafana.md:19]: not important - probably better show as placeholder

File: docs/metrics-via-grafana.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      password: {password}
+    http_sd_configs:
+      - url: https://cloud.mongodb.com/prometheus/v1.0/groups/{group-ID}/discovery
+        refresh_interval: 60s
+        basic_auth:
+          username: user
```
Comment (leo-ri) [inline docs/metrics-via-grafana.md:25]: ^

**Review by leo-ri — APPROVED** [review]

File: docs/metrics-via-grafana.md (hunk 1 above)
Comment (fabritsius) [reply docs/metrics-via-grafana.md:19]: Ok, great catch 👍

**Review by igor-karpukhin — APPROVED** [review]
Looks good 👍

**Pull request merged**